site/en/docs/extensions/mv3/storage-and-cookies/index.md

```diff
@@ -0,0 +1,116 @@
+
+Some web storage APIs like [IndexedDB][indexeddb] are accessible in service workers. However,
+[Local Storage][local-storage] and [Session Storage][session-storage] are not.
+
+If you have a specific use case, use an [offscreen document][offscreen] to access these.
+
+### Partitioning {: #storage-partitioning }
+
+Partitioning is where keys are introduced for stored data to limit where it can be accessed.
+
+When enabled, [storage partitioning][storage-partitioning] introduces changes to how partitioning
+keys are defined to prevent certain types of cross-site tracking. In practice, this means that if
+site A embeds an iframe containing site B, site B will not be able to access the same storage it
+would usually have when navigated to directly.
+
+To mitigate the impact of this in extensions, two key exemptions apply:
+
+- If a page with the `chrome-extension://` scheme is embedded in any site, storage partitioning will
+not apply, and the extension will have access to its top-level partition.
+- If a page with the `chrome-extension://` scheme includes an iframe, and the extension has
+[host permissions][declare-permissions] for the site it is embedding, that site will also have
+access to its top-level partition.
+
+## Cookies {: #cookies }
+
+Cookies provide a way to store key/value pairs associated with a specific domain and path. They have
+limited value in extensions but understanding their behavior can be useful if you have a specific
+use case or have bundled a third-party script which uses them in its implementation.
+
+### Secure cookies {: #secure-cookies }
+
+The [`Secure`][cookies-restrict-access] cookie attribute is only supported for the `https://`
+scheme. Consequently, `chrome-extension://` pages are not able to set cookies with this attribute.
+
+This also means that extension pages cannot use other cookie attributes where the `Secure` attribute is
+required:
+
+- [`SameSite=None`][same-site]
+- [`Partitioned`][chips]
+
+### Partitioning {: #cookies-partitioning }
+
+{% Aside %}
+See [https://crbug.com/1463991](https://crbug.com/1463991) which tracks possible changes to the
+behaviour when an extension with host permissions embeds a third-party site.
+{% endAside %}
+
+As mentioned [above](#secure-cookies), extensions cannot create partitioned cookies, and cookies
+are not currently partitioned by default (see the Privacy Sandbox
+[timeline][privacy-sandbox-timeline] for related work). Consequently, `chrome-extension://` pages
+embedded in other websites can access cookies associated with the extension origin as normal.
+
+If a `chrome-extension://` page contains an iframe, any sites it embeds will use the extension
+origin as the partitioning key. This means that they will **not** be able to access partitioned
+cookies associated with other partitions.
+
+The [`chrome.cookies`][chrome-cookies] API currently operates on cookies from all partitions. For
+more information, see the [API reference][chrome-cookies-partitioning].
```

Comment:
This is fine. A simpler way to do this is to use only the first sentence and put the partitioning link on 'cookies from all partitions'.